changecalculator/src/main/java/org/msstate/cse/changecalculator/database/DbUtil.java

```diff
@@ -0,0 +1,75 @@
+package org.msstate.cse.changecalculator.database;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+import org.hibernate.Session;
+import org.hibernate.Transaction;
+import org.msstate.cse.changecalculator.hibernate.HibernateUtil;
+import org.msstate.cse.changecalculator.hibernate.model.SvnHistroy;
+import org.msstate.cse.changecalculator.utility.Util;
+
+public class DbUtil {
+	private static PreparedStatement ps = null;
+	private static String tableName = null;
+	
+	
+	
+	public static void setTableName(String tableName) {

```

Comment:
This should be written as this.tableName = tableName since it is referring to itself.